### PR TITLE
Correct rules considered case-insensitive in dates

### LINKED ISSUE #473: Date case sensitivity
Caching 4.2:

> Although all date formats are specified to be case-sensitive, a cache recipient should match day, week, and time-zone names case-insensitively.

What does "week" refer to in this text? Also, why is this separate from the text on HTTP-dates? Do we really mean that caches should parse them in a different fashion than other components?

### PR DESCRIPTION
**Pull request by mnot**
And mention in Semantics.

Fixes #473.

### CODE REVIEW

**Review by reschke — COMMENTED** [review]
will add some links...

File: draft-ietf-httpbis-cache-latest.xml [hunk 1]
```diff
@@ -681,7 +681,7 @@
 </t>
 <ul>
    <li>Although all date formats are specified to be case-sensitive,
```
Comment (reschke) [inline draft-ietf-httpbis-cache-latest.xml:683]: AFAIU, we now list all parts of the ABNF that are not digits or delimiters. So why not simply match the whole string case-insensitively?

**Review by reschke — CHANGES_REQUESTED** [review]

File: draft-ietf-httpbis-cache-latest.xml (hunk 1 above)
Comment (mnot) [reply draft-ietf-httpbis-cache-latest.xml:683]: Fair point.

File: draft-ietf-httpbis-cache-latest.xml [hunk 2]
```diff
@@ -681,7 +681,7 @@
 </t>
 <ul>
    <li>Although all date formats are specified to be case-sensitive, 
-      a cache recipient &SHOULD; match day, week, and time-zone names
+      a cache recipient &SHOULD; match the day-name, day-name1, month and GMT rules
```
Comment (reschke) [inline draft-ietf-httpbis-cache-latest.xml:684]:
```suggestion
      a cache recipient &SHOULD; match the field value
```

**Review by reschke — APPROVED** [review]

**Pull request merged**